test/parallel/test-eslint-no-unescaped-regexp-dot.js

```diff
@@ -17,12 +17,14 @@ new RuleTester().run('no-unescaped-regexp-dot', rule, {
   ],
   invalid: [
     {
-      code: '/./',
-      errors: [{ message: 'Unescaped dot character in regular expression' }]
+      code: String.raw`/./`,
+      errors: [{ message: 'Unescaped dot character in regular expression' }],
+      output: String.raw`/\./`
     },
     {
       code: String.raw`/\\./`,
-      errors: [{ message: 'Unescaped dot character in regular expression' }]
+      errors: [{ message: 'Unescaped dot character in regular expression' }],
+      output: String.raw`/\./`
```

Comment:
This seems wrong... ? I believe it should be more like:

```js
String.raw`/\\\./`
```

The `\\` is part of the RegExp.

Reply:
Updated the logic and test file.